src/globus_cli/commands/group/member/add.py

```diff
@@ -0,0 +1,45 @@
+import click
+
+from globus_cli.login_manager import LoginManager
+from globus_cli.parsing import IdentityType, ParsedIdentity, command
+from globus_cli.termio import FORMAT_TEXT_RECORD, formatted_print
+from globus_cli.types import FIELD_LIST_T
+
+
+def _format_added_user_id(x: dict):
+    return x["add"][0]["identity_id"]
+
+
+def _format_added_user_username(x: dict):
+    return x["add"][0]["username"]
+
+
+def _format_group_id(x: dict):
+    return x["add"][0]["group_id"]
+
+
+ADD_USER_FIELDS: FIELD_LIST_T = [
+    ("Group ID", _format_group_id),
+    ("Added User ID", _format_added_user_id),
+    ("Added User Username", _format_added_user_username),
+]
+
+
+@command("add", short_help="Add one or more members to a group")
+@click.argument("group", type=str)
```

Comment:
Because it's a positional argument (and therefore has no help string), I think this should be `group_id`.
It's a small thing, but I think it will be better for users, since it clarifies that you need the ID, not the name or some other field here.

I would also type it as a `click.UUID` so that non-UUID inputs fail.